packages/synapse-react-client/src/components/Evaluation/EvaluationCard.tsx

```diff
@@ -191,19 +191,37 @@ function EvaluationCardDropdown({
         slotProps={{ paper: { sx: { minWidth: '120px' } } }}
       >
         {permissions.canEdit && (
```

Comment:
Fix a bug in the EvaluationCard (SWC challenge editor)  where menu options did not close after click